src/components/review/review.tsx

```diff
@@ -40,19 +85,20 @@ export default function Review ():JSX.Element {
       </div>
       <textarea
         className="reviews__textarea form__textarea"
-        id="review" value={reviewText}
-        onChange={(evt) => setReviewText(evt.target.value)}
+        id="review" value={reviewForm.comment}
+        onChange={(evt) => setReviewForm({...reviewForm, comment: evt.target.value})}
         name="review"
         placeholder="Tell how was your stay, what you like and what can be improved"
-        minLength={ReviewValidationParameters.MinLength}
-        maxLength={ReviewValidationParameters.MaxLength}
+        minLength={FormValidation.MinLength}
+        maxLength={FormValidation.MaxLength}
+        disabled={isBlockedForm}
       >
       </textarea>
       <div className="reviews__button-wrapper">
         <p className="reviews__help">
           To submit review please make sure to set <span className="reviews__star">rating</span> and describe your stay with at least <b className="reviews__text-amount">50 characters</b>.
         </p>
-        <button className="reviews__submit form__submit button" type="submit" disabled>Submit</button>
+        <button className="reviews__submit form__submit button" type="submit" disabled={!isValidForm(reviewForm.comment, reviewForm.rating) || isBlockedForm}>Submit</button>
```

Comment:
`const isButtonDisabled = isFormValid && !isPending(status)`

```suggestion
        <button className="reviews__submit form__submit button" type="submit" disabled={isButtonDisabled}>Submit</button>
```